Doc/library/types.rst

```diff
@@ -486,7 +486,7 @@ Coroutine Utility Functions
    The generator-based coroutine is still a :term:`generator iterator`,
    but is also considered to be a :term:`coroutine` object and is
    :term:`awaitable`.  However, it may not necessarily implement
-   the :meth:`__await__` method.
+   the __await__ method.
```

Comment:
I think the intent here is to link to [the entry for `__await__` in the data model](https://docs.python.org/3/reference/datamodel.html#object.__await__) (and I think that would be useful), in which case the syntax needs to be as follows

```suggestion
   the :meth:`~object.__await__` method.
```